src/components/AddressResolver/AddressesTable.tsx

```diff
@@ -202,77 +82,68 @@ export const AddressesTable: FunctionComponent<AddressesTableProps> = ({
   };
 
   return (
-    <div className={`${className} flex py-6`}>
-      <div className="flex w-full">
-        <StyledTableAddressResolver className="table-responsive">
-          <table className="table">
-            <thead className="table-thead">
-              <tr>
-                <th />
-                <td>Order</td>
-                <td>Name</td>
-                <td>Endpoint</td>
-                <td>API Header</td>
-                <td>API Key</td>
-                <td>&nbsp;</td>
-              </tr>
-            </thead>
-            <tbody className="table-tbody">
-              {thirdPartyAPIEndpoints.length === 0 && !isNewEndpoint && (
-                <tr>
-                  <th />
-                  <td>&mdash;</td>
-                  <td>&mdash;</td>
-                  <td>No third party&apos;s endpoint found.</td>
-                  <td>&nbsp;</td>
-                  <td>&nbsp;</td>
-                  <td>&nbsp;</td>
-                </tr>
-              )}
-              {thirdPartyAPIEndpoints.map((item, index) => {
-                const orderNumber = index + 1;
-
-                return (
-                  <EndpointEntry
-                    key={item.endpoint}
-                    orderNumber={orderNumber}
-                    isEndpointUrlExists={isCurrentEndpointUrlExists(item.endpoint)}
-                    removeEndpoint={() => {
-                      onRemoveEndpoint(item.name, index);
-                    }}
-                    onMoveEntryUp={() => {
-                      moveEntryUp(index);
-                    }}
-                    onMoveEntryDown={() => {
-                      moveEntryDown(index);
-                    }}
-                    onUpdateEndpoint={onUpdateEndpoint(index)}
-                    api={thirdPartyAPIEndpoints[index].endpoint}
-                    name={thirdPartyAPIEndpoints[index].name}
-                    apiHeader={thirdPartyAPIEndpoints[index].apiHeader}
-                    apiKey={thirdPartyAPIEndpoints[index].apiKey}
-                    canEdit={false}
-                  />
-                );
-              })}
-              {isNewEndpoint && (
-                <EndpointEntry
-                  orderNumber={thirdPartyAPIEndpoints.length + 1}
-                  isEndpointUrlExists={isEndpointUrlExists}
-                  removeEndpoint={() => {
-                    setNewEndpoint(false);
-                  }}
-                  onUpdateEndpoint={addNewEndpoint}
-                  api=""
-                  name=""
-                  apiHeader=""
-                  apiKey=""
-                  canEdit={true}
-                />
-              )}
-            </tbody>
-          </table>
-        </StyledTableAddressResolver>
+    <div className="flex md:py-6">
```

Comment:
don't think u need flex here, then the `md:py-6` probably can merge with the parent to become `md:py-12` or something

Reply:
done.